src/net.h

```diff
@@ -969,6 +969,12 @@ class CConnman
     void RecordBytesRecv(uint64_t bytes);
     void RecordBytesSent(uint64_t bytes) EXCLUSIVE_LOCKS_REQUIRED(!m_total_bytes_sent_mutex);
 
+    /**
+     Return networks for which we have no addresses in addrman and therefore
+     may require loading fixed seeds.
+     */
+    std::vector<Network> GetFixedSeedNetworks() const;
```

Comment:
I wish `for which we have no addresses` was captured by the function name somehow. Perhaps `GetRelevantFixedSeedNetworks`?

Reply:
Renamed it, pretty sure I even used that name in an earlier iteration.